### ISSUE TITLE
Refinement list trigger twice when clicked on custom template

### ISSUE THREAD

**Issue by Shipow** [algolia/instantsearch.js#299]
I'm using a refinementlist widget with a custom templates and it doesn't have the normal behavior (looks triggered twice) when I click on nested elements like the spans for value or count

```
templates:{
  item: '<a href="#" class="list-group-item{{#isRefined}} active{{/isRefined}}"><span class="value">{{name}}</span> <span class="badge pull-right">{{count}}</span></a>'
  }
```

![bug-refinementlist](https://cloud.githubusercontent.com/assets/976175/10612410/fa6de8aa-7750-11e5-845e-05624058baf5.gif)

**Comment by Shipow** [comment]
you can easilly reproduce it there: http://www.realtime-search.com/algolia/-JwRG-uarUOgOv3EPEhz

**Comment by bobylito** [comment]
Cool bug! Thanks :)

It occurs when someone click on a sub element of the link (here the count label). Remind me of the bug weeks ago when implementing the refinement list.

**Comment by pixelastic** [comment]
I actually thought that we fixed that in the refinementList. We'll have to test again.

**Comment by redox** [comment]
Probably already fixed in the `develop` branch.

**Comment by vvo** [comment]
@Shipow this should be fixed (we released 0.5.0), try again, even if your whole app can be broken
